idk/ingest.go

```diff
@@ -240,7 +244,17 @@ func (m *Main) Rename() {
 	}
 }
 
+// Basic sets up Main with basic functionality, excluding those things which are
+// not required for some implementations (such as the kafka runner in fbsql).
+func (m *Main) Basic() {
```

Comment:
rename to SetBasic? Just Basic makes it sound like a "getter"

Reply:
good call. I changed that and rebased everything.